src/System.Management.Automation/resources/Modules.resx

```diff
@@ -627,4 +627,49 @@
   <data name="CannotFindCoreCompatibleBuiltInModule" xml:space="preserve">
     <value>Cannot find the built-in module '{0}' that is compatible with the 'Core' edition. Please make sure the PowerShell built-in modules are available. They usually come with the PowerShell package under the $PSHOME module path, and are required for PowerShell to function properly.</value>
   </data>
+  <data name="WDACExportModuleCommandLogTitle" xml:space="preserve">
+    <value>Export-ModuleMember Cmdlet</value>
+  </data>
+  <data name="WDACExportModuleCommandLogMessage" xml:space="preserve">
+    <value>Export of module members will fail because module '{0}', has a different language mode '{1}' than the current session '{2}'.</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogTitle" xml:space="preserve">
+    <value>Module Implicit Function Export</value>
+  </data>
+  <data name="WDACImplicitFunctionExportLogMessage" xml:space="preserve">
+    <value>Implicit function export for module '{0}' will be denied, because it is trusted but the session is in ConstrainedLanguage mode (not trusted).</value>
```

Comment:
I'm not sure I understand the logic of this error. If it is trusted, why can it be exported.

Reply:
Implicit function export is not allowed across trust boundaries.  Instead you have to explicitly export functions you mean to be public.  This is to avoid inadvertently exposing dangerous functions.

Reply:
PowerShell will export all module functions when an explicit export is absent (legacy behavior).  But this behavior is not safe and is restricted in CL mode.

I updated this message but it is difficult to provide all needed context in a single event message.  So these events also include a FQID that I intended to eventually be used to link to more detailed information in our documents (e.g., 'FQID: ExportModuleMemberCmdletNotAllowed')).

Reply:
OK, that makes sense. I understand the need to keep the messages succinct.